src/errors/validation-error.ts

```diff
@@ -21,9 +21,14 @@ export class ValidationError extends Error {
     this.name = 'ValidationError';
   }
 
-  private buildError(err: any) {
+  /* eslint-disable sonarjs/cognitive-complexity */
+  private buildError(err: any, filepath: any) {
```

Comment:
```suggestion
  private buildError(err: any, filePath: string) {
```

Reply:
why string? passed value can be undefined, what will happen then?

Reply:
So please use optional argument. `filePath?: string`